src/main/java/net/sf/jabref/gui/dbproperties/SaveActionsPanel.java

```diff
@@ -138,19 +134,22 @@ public void storeSettings(MetaData metaData) {
     }
 
     public boolean hasChanged() {
-        List<FieldFormatterCleanup> newActions = ((SaveActionsListModel) actionsList.getModel()).getAllActions();
+        List<FieldFormatterCleanup> newActions = ((SaveActionsListModel<FieldFormatterCleanup>) actionsList.getModel())
```

Comment:
In this case it's necessary, it can't be derived automatically.
Previously all references to the type where non-generic